src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -309,5 +314,10 @@
                 <x:String x:Key="CaptionButtonPath">M 0 0 L 10 10 M 10 0 L 0 10</x:String>
             </ResourceDictionary>
         </Button.Resources>
+        <ToolTipService.ToolTip>
+            <ToolTip>
+                <TextBlock x:Uid="WindowCloseButtonToolTip" />

```

Comment:
Why is the Maximize one the odd duck out using a <Run> here that is named?

Reply:
The Maximize button needs to have it's tooltip text changed in code-behind depending on if it's "Maximize" or "Restore down"

Reply:
Yuck OK.